file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,551 @@
+      id                     TYPE c LENGTH 32,
+      "! <p class="shorttext">Object</p>
+      "! Object type
+      object                 TYPE c LENGTH 32,
+      "! <p class="shorttext">Type</p>
+      "! Object type
+      type                   TYPE c LENGTH 32,
+      "! <p class="shorttext">Parameter Structure ID</p>
+      "! Situation object structure id
+      parameter_structure_id TYPE c LENGTH 32,
+      texts                  TYPE ty_sit2_obj_event_t_list,
+    END OF ty_sit2_obj_events,
+
+    "! <p class="shorttext">Events</p>
+    "! Situation object events
+    ty_sit2_obj_events_list TYPE STANDARD TABLE OF ty_sit2_obj_events WITH DEFAULT KEY,
+
+    "! <p class="shorttext">End User Actions</p>
+    "! Situation object actions
+    BEGIN OF ty_sit2_obj_act,
+      "! <p class="shorttext">Reusable</p>
+      "! Object reusable
+      reusable    TYPE abap_bool,
+      "! <p class="shorttext">Type</p>
+      "! Action type of situation
+      "! $required
+      type        TYPE ty_sit2_do_action_type,
+      "! <p class="shorttext">Scope</p>
+      "! Source
+      "! $required
+      scope       TYPE ty_sit2_do_scope,
+      "! <p class="shorttext">ID</p>
+      "! Action id
+      id          TYPE c LENGTH 32,
+      texts       TYPE ty_sit2_obj_act_t_list,
+      navigations TYPE ty_sit2_obj_act_nav_list,
+      callbacks   TYPE ty_sit2_obj_act_cb_list,
+    END OF ty_sit2_obj_act,
+
+    "! <p class="shorttext">End User Actions</p>
+    "! Situation object actions
+    ty_sit2_obj_act_list TYPE STANDARD TABLE OF ty_sit2_obj_act WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Navigations</p>
+    "! Situation object navigation
+    BEGIN OF ty_sit2_obj_nav,
+      "! <p class="shorttext">Reusable</p>
+      "! Object reusable
+      reusable               TYPE abap_bool,
+      "! <p class="shorttext">Scope</p>
+      "! Source
+      "! $required
+      scope                  TYPE ty_sit2_do_scope,
+      "! <p class="shorttext">ID</p>
+      "! Navigation id
+      id                     TYPE c LENGTH 32,
+      "! <p class="shorttext">Semantic Object Action</p>
+      "! Semantic object action
+      semantic_object_action TYPE c LENGTH 50,
+      "! <p class="shorttext">Semantic Object</p>
+      "! Semantic object
+      semantic_object        TYPE c LENGTH 50,
+      parameters             TYPE ty_sit2_obj_nav_par_list,
+    END OF ty_sit2_obj_nav,
+
+    "! <p class="shorttext">Navigations</p>
+    "! Situation object navigation
+    ty_sit2_obj_nav_list TYPE STANDARD TABLE OF ty_sit2_obj_nav WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Callbacks</p>
+    "! Situation object callback
+    BEGIN OF ty_sit2_obj_cb,
+      "! <p class="shorttext">Reusable</p>
+      "! Object reusable
+      reusable   TYPE abap_bool,
+      "! <p class="shorttext">Scope</p>
+      "! Source
+      "! $required
+      scope      TYPE ty_sit2_do_scope,
+      "! <p class="shorttext">Class Name</p>
+      "! Reference to
```

Comment:
Could there be a better description for `class_name`? I don't understand to oder from what the reference is